src/ctrl/BoneEditor.cpp

```diff
@@ -143,7 +143,7 @@ void BoneEditor::resetCurrentTarget(QString* aMessage)
             mTarget.node = nullptr;
             if (aMessage)
             {
-                *aMessage = UILog::tr("The object which has an invalid posture was given.");
+                *aMessage = UILog::tr("An object with an invalid pose has been given");
```

Comment:
I was unsure between the differences regarding pose and posture in the code, since as far as I remember, the original Japanese translation mentioned both and I couldn't understand the entirety of the code snippet to decide on the better one myself so I took it from inference and context, that's the reason as to why it is inconsistent, I would love some feedback on this.